test/new-e2e/pkg/utils/clients/ssh.go

```diff
@@ -78,3 +80,80 @@ func ExecuteCommand(client *ssh.Client, command string) (string, error) {
 
 	return string(stdout), err
 }
+
+// CopyFile create a sftp session and copy a single file to the remote host through SSH
+func CopyFile(client *ssh.Client, src string, dst string) error {
+	sftpClient, err := sftp.NewClient(client)
+
+	if err != nil {
+		return err
+	}
+
+	if err := copyFile(sftpClient, src, dst); err != nil {
+		return err
+	}
+	return nil
+}
+
+// CopyFolder create a sftp session and copy a folder to remote host through SSH
+func CopyFolder(client *ssh.Client, srcFolder string, dstFolder string) error {
+
+	sftpClient, err := sftp.NewClient(client)
+	if err != nil {
+		return err
+	}
+
+	if err := sftpClient.MkdirAll(dstFolder); err != nil {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
What is the behaviour if the directory exists ?